tests/conftest.py

```
@@ -0,0 +1,103 @@
+import httpx
+import pytest
+
+from ml.models import BaseModelConfig, DataConfig, ModelConfig, TrainingConfig
+
+API_PORT = 8011
+
+
+@pytest.fixture(scope="module")
+def server():
+    repo_root = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    proc = subprocess.Popen(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)